config/ext/rspec.yml

```diff
@@ -543,6 +545,13 @@ RSpec/InstanceVariable:
   StyleGuide: https://rspec.rubystyle.guide/#instance-variables
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/InstanceVariable
 
+RSpec/IsExpectedSpecify:
```

Comment:
Hab hier keine große Meinung. Hab ich aber auch noch nie bei uns gesehen.
https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecexpectinlet :+1:

Reply:
I think we usually just use `it` in all cases so I don't have a strong opinion here either.

Reply:
I would never expect a `let` to contain expectations. Voting for enabling.